pages/osx/screencapture.md

```diff
@@ -0,0 +1,27 @@
+# screencapture
+
+> Utility to take screenshots and screen recordings.
```

Comment:
It would be nice if we could find a link for this tool.

Reply:
It's a builtin utility in macOS. I haven't been able to find the manpage on Apple's docs website, and none of the other builtins seems to have such a link, so I left it out.

Reply:
Oh. Ok then :)